src/work-allocation-2/containers/work-case-list/work-case-list.component.ts

```diff
@@ -0,0 +1,225 @@
+import { Component, EventEmitter, Input, OnChanges, Output } from '@angular/core';
+import { Router } from '@angular/router';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { PaginationParameter } from 'src/work-allocation-2/models/dtos';
```

Comment:
We shouldn't import from `src/`. This needs to be a relative path.

Reply:
Made relative.